IBX-1335: Rebranded Ibexa DoctrineSchema Bundle

alongosz opened the pull request:
| Question                                  | Answer
| ---------------------------------------- | ------------------
| **JIRA issue**                          | [IBX-1335](https://issues.ibexa.co/browse/IBX-1335)
| **Type**                                   | feature
| **Target Ibexa version** | `v4.0`
| **BC breaks**                          | yes

This PR contains rebranding of Ibexa Doctrine Schema Bundle (from `ezsystems/doctrine-dbal-schema`). It also fixes a few standard things either previously missed or outdated:

- Added GitHub Pull Request template,
- Forced PHP 7.4 or higher,
- Upgraded Ibexa Code Style to v2.0.

#### Postponed part

- [ ] `ez_doctrine_schema` extension alias and configuration tree root cannot be changed yet to `ibexa_doctrine_schema` because dependency on other packages (installer) and no BC for that part yet.

#### BC Documentation

~What stands out is the change of `ez_doctrine_schema` configuration key and Symfony DIC parameter to `ibexa.*`~

No package-specific BC yet to document, needed to be postponed.


#### TODO
- [x] Code style
- [x] Composer `replaces` config
- [x] `ez_doctrine_schema` -> `ibexa_doctrine_schema`
- [x] `ez.schema.build_schema` -> `ibexa.schema.build_schema`
- [x] Fix Installer issue

#### Checklist:
- [x] Provided PR description.
- [x] Tested the solution manually.
- [x] Checked that target branch is set correctly (master for features, the oldest supported for bugs).
- [x] Ran PHP CS Fixer for new PHP code (use `$ composer fix-cs`).
- [ ] Asked for a review (ping `@ibexa/engineering`).

Steveb-p commented on README.md:
```diff
@@ -50,9 +45,6 @@ class BuildSchemaSubscriber implements EventSubscriberInterface
-    /**
-     * @param \EzSystems\DoctrineSchema\API\Builder\SchemaBuilderEvent $event
-     */
     public function onBuildSchema(SchemaBuilderEvent $event)
```
Since `importSchema` received returned type, shouldn't this method do so as well?

alongosz replied:
Ah, yes, I shouldn't modify return type at all, it's a bit out of scope. Can we postpone it till later?

Steveb-p replied:
I mean you've added it to the other method :smile: So I'd either add the types, or not modify that at all.

But :+1: nonetheless.

adamwojs commented on .php-cs-fixer.php:
```diff
@@ -0,0 +1,17 @@
+
+use Ibexa\CodeStyle\PhpCsFixer\InternalConfigFactory;
+
+$finder = PhpCsFixer\Finder::create()
+    ->in(__DIR__)
+    ->files()->name('*.php');
```
I know that internally implementation will exclude `vendor` directory  but I would really prefer to explicit list `src` and `tests` directories.

adamwojs commented on .travis.yml:
```diff
@@ -11,11 +11,11 @@ cache:
           env: CMD='composer test' MYSQL_DATABASE_URL='mysql://root@localhost/testdb'
         - name: 'Code Style validation'
-          php: '7.3'
+          php: '7.4'
           env: CMD='php ./vendor/bin/php-cs-fixer fix --dry-run -v --show-progress=estimating'
```
```
  [PhpCsFixer\ConfigurationException\InvalidConfigurationException (16)]        
  The progress type "estimating" is not defined, supported are "none", "dots".
```

alongosz commented:
@Steveb-p @adamwojs I'm sorry, I missed your remarks. When merging it on my screen there was no more messages here. Must be GitHub http-cache issue. Fixed in #2. Travis is not used for `ibexa/*` so I replaced it with GitHub Actions
